orange/team-has-a-security-champion.md

```diff
@@ -1,13 +1,18 @@
 # Team has a Security Champion
 
-Security Champions bring security knowledge to teams so that they can take responsibility for security.
+Find and train a [Security Champion](https://github.com/AppSecure-nrw/security-belts#security-champion) from within the team.
 
-## Risk
+*Note*: The Security Champion should be a volunteer interested in software security. Just forcing someone to take the role is worse than having none.
 
-- No one feels directly responsible for security.
-- Central security experts do not have enough time to support each team.
+## Benefits
+
+- The Security Champion brings security knowledge to the team so that the team can take responsibility for security.
```

Comment:
```suggestion
- The Security Champion brings security knowledge to the team so that the team can make informed security decisions and take responsibility for security.
```